bundles/org.openhab.binding.solarman/src/main/java/org/openhab/binding/solarman/internal/modbus/SolarmanProtocolFactory.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.solarman.internal.modbus;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.solarman.internal.SolarmanLoggerConfiguration;
+
+/**
+ * @author Peter Kretz - Added RAW Modbus for LAN Stick
+ */
+@NonNullByDefault
+public class SolarmanProtocolFactory {
+
+    public static SolarmanProtocol CreateSolarmanProtocol(SolarmanLoggerConfiguration solarmanLoggerConfiguration) {
```

Comment:
Probably a typo: methods should start with a lower case.

Reply:
Fixed it

Reply:
I updated the [README.md](https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.solarman/README.md) explaining the new option and added LSE-3 logger as new logger benfitting from this new option.